source/includes/fact-system-vars.rst

```diff
@@ -0,0 +1,108 @@
+.. list-table::
+   :widths: 30 10 40 10 10
+   :stub-columns: 1
+   :header-rows: 1
+
+   * - System Variable
+     - Data Type
+     - Description
+     - Default Value (Atlas-hosted)
+     - Default Value (On Premises)
+   * - ``full_pushdown_exec_mode``
+     - boolean
+     - Specifies whether a query error is returned for queries with
+       clauses that aren't fully pushed down. SQL query predicates
+       usually appear in ``WHERE`` clauses and are used to filter data
+       returned by the query.
+     - ``0``
+     - ``0``
+   * - ``log_level``
+     - integer
+     - Specifies the logging level for |bi-short|:
+
+       - ``-1``: Do not log.
+       - ``0``: Log only messages for database users, such as basic
+         ``mongosqld`` events and state changes.
+       - ``1``: Log messages for database users and ``mongosqld`` administrators.
+       - ``2``: For internal use only.
+     - ``2``
+     - ``0``
+   * - ``max_nested_table_depth``
+     - integer
+     - Specifies the maximum number of unique nested field paths that
+       ``mongosqld`` maps to a relational table for a collection.
+     - ``50``
+     - ``50``
+   * - ``max_num_columns_per_table``
+     - integer
+     - The maximum number of unique fields that ``mongosqld`` maps to
+       relational columns for a collection.
+     - ``1000``
+     - ``2000``
+   * - ``mongodb_max_varchar_length``
+     - integer
+     - Specifies the maximum string length returned for columns using
+       the ``VARCHAR`` data type. A value of ``0`` specifies
+       no limit.
+     - ``0``
+     - ``0``
+   * - ``polymorphic_type_conversion_mode``
+     - string
+     - Determines how MongoDB evaluates document fields that are
+       specified with multiple data types. For example, ``count``
+       could be a number in one document and a string in another.
+
+       - ``off``:
+         Queries may fail if you do not explicitly cast
+         document fields that are specified with multiple data types.
+       - ``fast``:
+         |bi-short| casts document fields as appropriate
+         if MongoDB samples at least two documents during schema
+         generation that use different data types for the same field.
+         In the example, MongoDB would cast ``count`` as an integer.
```

Comment:
Actually, the type _mongosqld_ would cast `count` to, depends on the `schema_mapping_mode` setting. With `majority`, it would pick the majority sampled type. With `lattice`, it would always pick `string` (in your example). 

The main function of the `fast` mode is to automatically convert _only_ those fields (columns) that presented polymorphic data types during sampling. Whereas in `safe` mode, the BIC will unconditionally convert _all_ fields to the sampled type (determined by the `schema_mapping_mode` variable) - regardless of whether the field presents polymorphic data types during sampling.

Reply:
Fixed.
Follow-up question: What happens if the user doesn't specify any value for schema_mapping_heuristic (I assume you meant this rather than schema_mapping_mode)? Is polymorphic_type_conversion_mode ignored if schema_mapping_hueristic is not specified?

Reply:
If the user doesn't specify any value for `schema_mapping_mode`, the default value - `lattice` is used.

`polymorphic_type_conversion_mode` is never ignored but is `off` by default.

Reply:
I went with the following to simplify: "The data type that |bi-short| selects depends on the value of the ``schema_mapping_mode`` system variable."  This is technically correct whether or not they set the that sys var.  It's simpler because I don't have to add a caveat that we use the default value of schema_mapping_mode if they don't set it.